app/views/calculator/payment_today_form.scala.html

```diff
@@ -68,15 +70,15 @@
             name = "amount",
             value = dataForm.data.get("amount"),
             hint = Some(Hint(
-                content = Text(messages("ssttp.calculator.form.payment_today.hint", currency(debits.map(_.amount).sum)))
+                content = Text(messages("ssttp.calculator.form.payment_today.hint", currency(debits.map(_.amount).sum - 2)))
```

Comment:
you could maybe read `2` from `CalculatorForm.MinLeftOverAfterUpfrontPayment` so that we have less to change if the value ever changes from `2`